src/Sentry.AspNet/HttpContextExtensions.cs

```diff
@@ -37,8 +64,9 @@ public static ITransaction StartSentryTransaction(this HttpContext httpContext)
     {
         var method = httpContext.Request.HttpMethod;
         var path = httpContext.Request.Path;
+        var options = SentrySdk.CurrentOptions;
```

Comment:
Can `CurrentOptions` return null? on Java and friends, we have a NoOpOptions in case there's nothing bound to the Hub or no Hub at all, so we don't need to deal with NPEs.
Since .NET has explicitly nullability, that should be fine.

Reply:
Yes, it is marked as `SentryOptions?` and can return null. It's passed down mostly for logging purposes, so in the case of null, just no debug logs will be written.  That would only happen in testing.